src/hooks/usePoolTickData.ts

```diff
@@ -162,9 +155,7 @@ function useTicksFromSubgraph(
         )
       : undefined
 
-  return useAllV3TicksQuery(poolAddress ? { poolAddress: poolAddress?.toLowerCase(), skip: 0 } : skipToken, {
-    pollingInterval: ms`30s`,
-  })
+  return useAllV3TicksQuery(poolAddress, 0)
```

Comment:
looks like we're loosing a polling feature here. Is there a straightforward way to preserve this?